content/updates/2017-06-12-release-notes.md

```diff
@@ -0,0 +1,24 @@
++++
+date = "2017-06-12"
+title = "Platform Release Notes: June 12, 2017"
++++
+
+Curious what’s new that you might find helpful as a cloud.gov application developer? Here are highlights from our platform updates over the past week and a half.
+<!--more-->
+
+### Changed
+
+- [Fixed IP addresses for communicating with external apps](https://cloud.gov/docs/apps/static-egress/#cloud-gov-egress-addresses): Our customers often want cloud.gov apps that can talk to the outside and we now have an enabler to make that happen in a more secure way. Outbound traffic from cloud.gov comes from specific IP addresses to help customers open up a connection between cloud.gov and something hosted elsewhere.
```

Comment:
This description may be confusing in a couple of small ways - in this case, "fixed" means "static", but in a release notes post, "fixed" may read as "updated"! Also, cloud.gov apps can already talk to the outside, and this doesn't necessarily add substantial security (depending on context), but this change does enable better compatibility with common agency practices.